### PR TITLE
[RW-7339][risk=no] New e2e workspace writer notebook actions test

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

**Review by yonghaoy — APPROVED** [review]

File: e2e/package.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
     "test-nightly:debug": "cross-env PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false WORKBENCH_ENV=test TEST_MODE=nightly-integration yarn _test --detectOpenHandles",
-    "test-nightly-staging:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false TEST_MODE=nightly-integration yarn _test --maxWorkers=1",
     "test-local-nightly": "cross-env WORKBENCH_ENV=local TEST_MODE=nightly-integration yarn _test --maxWorkers=3",
     "test-local-nightly:debug": "cross-env PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false WORKBENCH_ENV=local TEST_MODE=nightly-integration yarn _test --detectOpenHandles",
     "test-perf": "cross-env WORKBENCH_ENV=perf yarn _test",
     "test-perf:debug": "cross-env PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false WORKBENCH_ENV=perf yarn _test --maxWorkers=1",
     "test-staging": "cross-env WORKBENCH_ENV=staging yarn _test --maxWorkers=3",
     "test-staging:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false yarn _test --maxWorkers=1",
+    "test-staging-nightly": "cross-env WORKBENCH_ENV=staging TEST_MODE=nightly-integration yarn _test --maxWorkers=2",
```
[the remark this answers is not in the document]
Comment (aweng98) [reply e2e/package.json:27]: Unrelated. I'll remove from this PR.

File: e2e/resources/workbench-config.ts [hunk 2]
```diff
@@ -59,8 +59,8 @@ const staging: IEnvConfig = {
   READER_USER: process.env.READER_USER || 'puppetcistagingreader1@staging.fake-research-aou.org',
   ACCESS_TEST_USER: process.env.ACCESS_TEST_USER || 'puppet-access-tester-1@staging.fake-research-aou.org',
   ADMIN_TEST_USER: process.env.ADMIN_TEST_USER || 'puppeteer-admin-1@staging.fake-research-aou.org',
-  DEFAULT_CDR_VERSION_NAME: 'Synthetic Dataset v4',
-  ALTERNATIVE_CDR_VERSION_NAME: 'Synthetic Dataset v3',
+  DEFAULT_CDR_VERSION_NAME: 'Synthetic Dataset v3',
```
[the remark this answers is not in the document]
Comment (aweng98) [reply e2e/resources/workbench-config.ts:62]: Also unrelated to new test. I saw it was wrong so just went ahead fixed it.

File: e2e/tests/notebook/notebook-writer-actions.spec.ts [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        codeFile: 'resources/python-code/gsutil.py',
+        markdownWorkaround: true
+      })
+    ).toMatch(/success$/);
+
+    await notebook.save();
+    await signOut(page);
+  });
+
+  // TODO (RW-7312): Re-enable once Terra clone is fixed.
```
[the remark this answers is not in the document]
Comment (aweng98) [reply e2e/tests/notebook/notebook-writer-actions.spec.ts:52]: Yes, test is not working due to that issue.

**Review by aweng98 — COMMENTED** [review]
Thank you.

File: e2e/resources/workbench-config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (calbach) [reply e2e/resources/workbench-config.ts:62]: v3 is an old version in staging, this is not correct. Not sure whether v4 worked or not - will try.

the WGS version is "Synthetic Dataset in the Controlled Tier", it's in the controlled tier section though; not sure if puppeteer can utilize that dropdown or not

**Pull request merged**